### ISSUE TITLE
working on release osTicket (v1.14.1) ?

### ISSUE THREAD

**Issue by eddogustian** [clonemeagain/plugin-autocloser#29]
hello, everyone..
working on release osTicket (v1.14.1) ?

your response is very meaningful, Thank you :)

**Comment by mmiat** [comment]
is this project abandoned?

**Comment by clonemeagain** [comment]
We can just abandon them? Nice

I mean, I didn't intend to abandon it, but I'm so busy that even answering email is hard to find time for, happy to accept pull requests and assistance/maintainer/takeover whatever.. but dunno man, osticket seem to hate my ideas and break my plugins on purpose, so I've given up on new ones.

**Comment by mmiat** [comment]
soory to read this, but I understand your point of view. thanks for reply.

**Comment by elekgeek** [comment]
ignore them, please make it work for 1.14.x because we really need this plugin, thank you for your efforts

**Comment by SoporteMesi** [comment]
> 
> 
> ignore them, please make it work for 1.14.x because we really need this plugin, thank you for your efforts

I agree with elekgeek. It's a pity to loose this plugins and it would be wonderful to have it working for 1.14.
Thanks for creating it.

**Comment by ililminati** [comment]
Good night, it is a shame this friction between you and the osTicket developers. I'm deploying osTicket now and your plugin would be very welcome, however, your indignation is understandable. I hope to see you updating your plugin soon.

**Comment by clonemeagain** [comment]
Plugin updated to support 1.18.
